src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/HttpApi/ApiClientExtensions.cs

```diff
@@ -429,6 +429,11 @@ public static async Task<List<OperationSummary>> GetOperations(this ApiClient cl
             return await client.GetOperations(tags, timeoutSource.Token).ConfigureAwait(false);
         }
 
+        public static Task<HttpStatusCode> StopEgressOperation(this ApiClient client, Guid operationId)
+        {
+            return StopEgressOperation(client, new Uri(client.BaseAddress, FormattableString.Invariant($"operations/{operationId}")));
+        }
+
         public static async Task<HttpStatusCode> StopEgressOperation(this ApiClient client, Uri operation)
```

Comment:
/todo Refactor ApiClient operation methods to use identifiers rather than raw URIs